test/test_private.py

```diff
@@ -54,51 +55,51 @@ def test_parse_rfc3339(self):
         nzdt = datetime.timezone(datetime.timedelta(hours=13))
         utc = datetime.timezone.utc
 
-        self.assertEqual(timeconv.parse_rfc3339('2020-12-25T13:45:50+13:00'),
-                         datetime.datetime(2020, 12, 25, 13, 45, 50, 0, tzinfo=nzdt))
+        assert timeconv.parse_rfc3339('2020-12-25T13:45:50+13:00') == \
```

Comment:
I'm pretty sure ruff format will not like this, and we might as well avoid making more work for ourselves later.

I think ruff will want this in parentheses instead, which is probably a whole discussion about readability in itself.

Reply:
What do we do here about it?

Reply:
This is an automated conversion by pytestify and for long lines, `assert` can seem uglier than `self.assertXxx` since the latter can break the line by the comma while the former will use a hard line break which seems less nice.

This can be fixed even if we increase the max line length to 120, which means some refactoring is required. For example: instead of writing this in one line, we can:

- `t1 = timeconv.parse_rfc3339('2020-12-25T13:45:50+13:00')`
- `t2 = datetime(2020, 12, 25, 13, 45, 50, 0, tzinfo=nzdt)`
- then `assert t1 == t2`

What do you think?

Reply:
(Sorry, the above comment was pending when you posted so probably reads oddly now that it's out of order).

I think we leave them for now and leave this as a decision for our discussion about potentially changing formatter.